test/TestUtilities/Test.Utility/Signing/OcspResponder.cs

```diff
@@ -89,7 +89,9 @@ public override void Respond(HttpListenerContext context)
                 var certificateId = request.GetCertID();
                 var certificateStatus = CertificateAuthority.GetStatus(certificateId);
                 var thisUpdate = _options.ThisUpdate ?? now;
-                var nextUpdate = _options.NextUpdate ?? now.AddSeconds(1);
+                //On Windows, if the current time is equal (to the second) to a notAfter time (or nextUpdate time), it's considered valid.
+                //But OpenSSL considers it already expired (that the expiry happened when the clock changed to this second)
+                var nextUpdate = _options.NextUpdate ?? now.AddSeconds(2);
```

Comment:
Does this mean that every test that calls `WaitForResponseExpirationAsync` will run 2 seconds slower?

Reply:
Yes, tests call `WaitForResponseExpirationAsync ` will be slower(1 second slower).  And there're 7 of them in total.